components/bikeShareChartMantine.tsx

```diff
@@ -0,0 +1,46 @@
+import { useFetchBikeShare } from "@/hooks/useFetchBikeShare";
+import { BarChart } from "@mantine/charts";
+
+const parseBikeSharesByState = (data) => {
+  const stateAggregatedData = data.features.reduce((acc, feature) => {
+    const state = feature.attributes.state;
+    if (!acc[state]) {
+      acc[state] = 0;
+    }
+    acc[state] += 1;
+    return acc;
+  }, {});
+  return Object.keys(stateAggregatedData)
+    .map((state) => {
+      return {
+        state,
+        count: stateAggregatedData[state],
+      };
+    })
+    .sort((a, b) => b.count - a.count);
+};
+
+const BikeShareChartMantine: React.FC = () => {
+  const { data } = useFetchBikeShare();
+  if (!data) {
+    return null;
+  }
+  const chartData = parseBikeSharesByState(data);
+  return (
+    <BarChart
+      h={575}
+      data={chartData}
+      dataKey="state"
+      tickLine="y"
+      orientation="vertical"
+      xAxisLabel="Count"
+      yAxisLabel="State"
+      xAxisProps={{ orientation: "top", height: 100}}
```

Comment:
these are all pass-through props to rechart -- but I don't think Mantine exposes all the options (i.e. they provide no way to pass through label props, only axis and bar props)

Reply:
dang, I wonder if we should just use recharts?

Reply:
I'll give it a shot and we can see how it looks

Reply:
ok -- new example up with rechart! the code is nice imho -- but I still ran into a lot of troubleshooting and had to dig pretty deep in the docs and github issues to get data rendering. I don't feel like I've hit the holy grail of easy and flexible with any of these libraries. I think my initial instinct of just mantine for now still feels true -- that was the fastest by far to get at least mostly working for me. You have more experience with the charting libraries though, what do you think?

Reply:
I think for the sake of this template, we should stick with Mantine charts. I think ideally this template has as low of a learning curve as possible. If prototypes need more complex charts they can always install Visx or recharts or whatever they choose to get their job done. But I think for this template we can recommend Mantine charts for simple implementations.

Reply:
Thank you for testing out recharts though!